minimal-shadow-cljs-nodejs/src/server/slack_events.cljs

```diff
@@ -0,0 +1,17 @@
+(ns server.slack-events
```

Comment:
Could this be bundled with the existing slack ns?

Reply:
yes - i think it may already be in a later pr